package.json

```diff
@@ -11,7 +12,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
:D Did you deploy?!

Reply:
Yes, I did. https://sophearychiv.github.io/litter-patrol/